### PR TITLE
Stop appending to the Rails assets load path.

### PR DESCRIPTION
**Pull request by lucasmazza**
As all shims/polyfills are required from the `app/assets/javascripts`, we don't
need to add the current `node_modules` directory to the load path.

### CODE REVIEW

File: lib/babel-schmooze-sprockets/engine.rb [hunk 1]
```diff
@@ -1,7 +1,4 @@
 module BabelSchmoozeSprockets
   class Railtie < Rails::Engine
-    initializer "babel-schmooze-sprockets" do
-      config.assets.paths << File.expand_path("#{__dir__}/../../node_modules")
-    end
   end
 end
```
Comment (rmacklin) [inline lib/babel-schmooze-sprockets/engine.rb:4]: Could we remove the whole file or does having an empty Railtie do something?

Comment (lucasmazza) [inline lib/babel-schmooze-sprockets/engine.rb:4]: as far as I remember it adds the `app/assets/javascripts` directory into the assets load path for us.

Comment (fnando) [inline lib/babel-schmooze-sprockets/engine.rb:4]: Actually, it's not a Railtie, but an Engine. And it adds tons of things to the load path, including what @lucasmazza mentioned. It also allows defining custom routes, migrations, and more.

I fixed the naming issue on 51701ddc5a17a418cab2178fa3267ea19b34a79b.

Comment (rmacklin) [inline lib/babel-schmooze-sprockets/engine.rb:4]: Oh, duh. Got confused. Thanks.

**Pull request merged**